src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -10,176 +10,318 @@ import expectJSX from 'expect-jsx';
 expect.extend(expectJSX);
 
 describe('RefinementList', () => {
-  let renderer;
-  let parentListProps;
-  let itemProps;
-
-  beforeEach(() => {
-    let {createRenderer} = TestUtils;
-    let cssClasses = {
-      list: 'list',
-      item: 'item',
-      active: 'active'
-    };
-    let templateData = {
-      cssClasses,
-      url: 'url'
-    };
-    let commonItemProps = {
-      handleClick: () => {},
-      itemClassName: 'item',
-      subItems: undefined,
-      templateKey: 'item',
-      templateProps: {}
+  function shallowRender(extraProps: {}) {
+    let props = {
+      createURL: () => 'url',
+      facetValues: [],
+      ...extraProps
     };
+    return shallow(React.createElement(RefinementList, props));
+  }
+
+  describe('cssClasses', () => {
+    it('should add the `list` class to the root element', () => {
+      // Given
+      let props = {
+        cssClasses: {
+          list: 'list'
+        }
+      };
+
+      // When
+      let actual = shallowRender(props);
+
+      // Then
+      expect(actual.hasClass('list')).toEqual(true);
```

Comment:
You can use toBe(true) (easier to read).

Equal means deepEqual, toBe means ===